### DISCUSSION TITLE
It doesn't launch on Windows.

### DISCUSSION THREAD

**Discussion by bangflash** [browser-use/browser-use#115] [Q&A]
"I'm trying to launch in a Windows Anaconda environment, but I'm encountering an error. Is this not supported?

The error message is:
ModuleNotFoundError: No module named 'ChatOpenAI'
and I can't proceed further."

**Comment by Louis454545** [comment]
You have to install package

**Reply by endur4ever** [reply]
as in "pip install -r requirements.txt" OR call out that package

**Reply by Louis454545** [reply]
You want use package or clone and use it ?

**Comment by dheerajgupta5680** [comment]
you have to install langchain openai explicitly by 'pip install langchain_openai'

**Reply by endur4ever** [reply]
@Louis454545 that's what I did as well. I would say look at the errors, after I did the pip install I noticed it couldn't find the module @dheerajgupta5680 mentioned.

**Reply by Louis454545** [reply]
> @Louis454545 that's what I did as well. I would say look at the errors, after I did the pip install I noticed it couldn't find the module @dheerajgupta5680 mentioned. 

Error please

**Reply by endur4ever** [reply]
correction* BEFORE the pip install of that module that was mentioned I was getting module not found error or equiv. Its working now for me. Also you may need to install playwright... I did.
